lib/publiccloud/gcp_client.pm

```diff
@@ -35,7 +35,7 @@ sub init {
     assert_script_run('source ~/.bashrc');
     if (is_sle('>=15')) {
         # kill it in case it's running
-        script_run("killall chronyd && sleep 5 && if pgrep chronyd; then killall -9 chronyd; fi");
+        script_run("systemctl stop chronyd.service && sleep 5 && if pgrep chronyd; then killall -9 chronyd; fi");
```

Comment:
```suggestion
        script_run("systemctl stop chronyd.service && sleep 5 && pgrep chronyd && killall -9 chronyd");
```

Reply:
Thank you for suggestion Maurizio but I'd like to skip it.
That `if` condition ensures that if `chronyd` is already dead then the command will return `0`.
Without that condition the result in case `chronyd` not running will be `1` and that's not nice :smile:

Please check my current version.